src/main/java/org/opentripplanner/transit/model/basic/TransitMode.java

```diff
@@ -29,6 +38,17 @@ public static Set<TransitMode> transitModesExceptAirplane() {
     return NO_AIRPLANE_MODES;
   }
 
+  /**
+   * This method returns the list of modes that are considered 'transit' by users, removing
+   * carpool.
+   */
+  public static TransitMode[] modesConsideredTransitByUsers() {
+    return Arrays
```

Comment:
You could use same syntax as transit no airplane?

Reply:
Good point. Corrected.